x-pack/plugins/ingest_manager/common/services/routes.ts

```diff
@@ -28,7 +28,7 @@ export const epmRouteService = {
   },
 
   getFilePath: (filePath: string) => {
-    return `${EPM_API_ROOT}${filePath}`;
+    return `${EPM_API_ROOT}${filePath.replace('/package', '/packages')}`;
```

Comment:
There is no way to fix the root cause of that?  where do we generate filePath for assets?

Reply:
The `/package` here is coming from the Registry. It returns paths relative to itself e.g.  `"assets": [ "/package/iptables-1.0.4/manifest.yml" ...]` on https://epr.elastic.co/package/iptables-1.0.4

We could (and may eventually) have EPM run a transform on the registry response before responding with EPM-relative routes like `"assets": [ "/packages/iptables-1.0.4/manifest.yml" ...]` but I went with this simple change which doesn't change the data flow or API responses and only makes changes inside the functions.

We'll continue to improve on how we deal with routes & relationships between services, but I think this is a cheap & safe win.

Reply:
or we can change EPR directories/path to use `package` instead of `packages` at some point?